libs/libc/math/lib_pow.c

```diff
@@ -41,6 +41,22 @@
 #ifdef CONFIG_HAVE_DOUBLE
 double pow(double b, double e)
 {
-  return exp(e * log(b));
+  if (b > 0)
```

Comment:
I'm not sure if that matters, but better to
```suggestion
  if (b > .0)
```
and same for line 48